src/translations/en.json

```
@@ -1966,10 +1967,12 @@
             "entities": {
               "name": "Entities",
               "show_header_toggle": "Show Header Toggle?",
-              "toggle": "Toggle entities."
+              "toggle": "Toggle entities.",
+              "description": "Entities will be the most common type of card that will also be the most familiar to people using the standard interface. It groups items together very close to how groups used to do."
```

Made it future tense.

I wonder if we should refer to the "old" UI / states UI... Since it is removed by the time this is released. New folks would not have a clue what you are talking about.

```suggestion
              "description": "Entities is the most common type of card that is the most familiar to people using the standard interface. It groups items together, very close to how groups used to do."
```

---

I agree. We should take the old UI reference out